lib/colorado_lottery.rb

```diff
@@ -0,0 +1,28 @@
+class ColoradoLottery
+  attr_reader :registered_contestants,
+              :winners,
+              :current_contestants
+
+  def initialize
+    @registered_contestants = Hash.new
+    @winners = []
+    @current_contestants = []
```

Comment:
Keep an eye on the interaction pattern - `@current_contestants` should be an empty hash.